documentation/docs/30-advanced/20-hooks.md

```diff
@@ -4,12 +4,13 @@ title: Hooks
 
 'Hooks' are app-wide functions you declare that SvelteKit will call in response to specific events, giving you fine-grained control over the framework's behaviour.
 
-There are two hooks files, both optional:
+There are three hooks files, all optional:
 
 - `src/hooks.server.js` — your app's server hooks
 - `src/hooks.client.js` — your app's client hooks
+- `src/hooks.js` — your app's isomorphic hooks
 
-Code in these modules will run when the application starts up, making them useful for initializing database clients and so on.
+Code in the client & server modules will run when the application starts up, making them useful for initializing database clients and so on.
```

Comment:
```suggestion
Code in the client and server modules will run when the application starts up, making them useful for initializing database clients and so on.
```

Reply:
Why does it apply to the client and server files, but not the isomorphic file? I think it'd be a bit more consistent if it applied to all